### DISCUSSION TITLE
Thoughts about updating readme?

### DISCUSSION THREAD

**Discussion by GalaxyShark** [ashishpawar517/AutoCompiler#26] [Q&A] [answered]
Any thoughts on updating README? new GIFs , Logos? anything from your end @ashishpawar517 

- for starting we can update the logo matching the github theme, white background logo looks kinda odd in Dark Github theme

**Comment by ashishpawar517** [comment, marked as answer]
I am working on it @GalaxyShark but no plans right now, working on some updating on the codebase. 

for the dark theme readme logo, you can add it, test it, raise PR, and review it.

**Reply by GalaxyShark** [reply]
okay

**Reply by ashishpawar517** [reply]
If you are working on AutoCompiler please consider working on a codebase other than README.

**Reply by GalaxyShark** [reply]
Actually, README attracts more contributors to github so that's why I asked. Sure, got your point.

**Reply by GalaxyShark** [reply]
Will consider working on this `RegexParser` ,will raise the PR

**Reply by ashishpawar517** [reply]
Great @GalaxyShark , check different the regex patterns
